src/goto-analyzer/static_verifier.cpp

```diff
@@ -20,48 +21,42 @@ Author: Martin Brain, martin.brain@cs.ox.ac.uk
 struct static_verifier_resultt
 {
   // clang-format off
-  enum { TRUE, FALSE, BOTTOM, UNKNOWN } status;
+  enum statust { TRUE, FALSE, BOTTOM, UNKNOWN } status;
   // clang-format on
   source_locationt source_location;
   irep_idt function_id;
 };
 
+/// Makes a status message string from a status.
+static std::string message(const static_verifier_resultt::statust &status)
```

Comment:
Suggest `const char *` since this only ever returns constants (no need to bind our caller to necessarily go through `std::string`)

Reply:
Done.